plugins/out_stackdriver/stackdriver_operation.c

```diff
@@ -0,0 +1,137 @@
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2019-2020 The Fluent Bit Authors
+ *  Copyright (C) 2015-2018 Treasure Data Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+#include "stackdriver.h"
+#include "stackdriver_operation.h"
+
+typedef enum {
+    NO_OPERATION = 1,
+    OPERATION_EXISTED = 2
+} operation_status;
+
+
+void add_operation_field(flb_sds_t *operation_id, flb_sds_t *operation_producer, 
+                                bool *operation_first, bool *operation_last, 
```

Comment:
align arguments to the start of the previous line where flb_sds_t  is located

Reply:
Done.